lib/mayaUsd/render/mayaToHydra/renderGlobals.cpp

```diff
@@ -177,10 +177,12 @@ void _CreateBoolAttribute(
     _CreateNumericAttribute<bool>(node, attrName, MFnNumericData::kBoolean, defValue);
 }
 
+#if USD_VERSION_NUM >= 2005
```

Comment:
Defined but not used warning as error I fixed along the way.  I guess I'm using an old USD...